lib/mapper-v1.js

```diff
@@ -34,7 +35,25 @@ exports.identify = function(msg, settings) {
   ret.user_id = msg.userId();
   ret.custom_attributes = traits;
 
-  if (msg.created()) ret.remote_created_at = time(msg.created());
+  var created = time(msg.created())
+
+  if (created) {
```

Comment:
would be great to get some comments on what the difference between `remote_created` and `created` is